contracts/test/ERC20Test.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@uma/core/contracts/common/implementation/TestnetERC20.sol";
+
+/**
+ * @notice Simulated ERC20 for use in testing bridge transfers.
+ */
+contract ERC20Test is TestnetERC20 {
+    constructor() TestnetERC20("ERC20 Test", "ERC20_TEST", 18) {} // solhint-disable-line no-empty-blocks
```

Comment:
Why is this contract being committed to this repo? You could just deploy an ERC20 with:

```
  const erc20 = await (await getContractFactory("ExpandedERC20", deployerWallet)).deploy("ERC20 Test", "ERC20_TEST", 18);
```

Reply:
Holdover from testing: removed https://github.com/across-protocol/contracts-v2/pull/381/commits/d5c2afaec63b225bd68c3ae941e1c34d15a29dca